site/source/docs/getting_started/FAQ.rst

```diff
@@ -587,35 +582,6 @@ with
    Module object, see :ref:`that FAQ entry<faq-export-stuff>`.
 
 
-Why do I get a ``NameError`` or ``a problem occurred in evaluating content after a "-s"`` when I use a ``-s`` option?
-=====================================================================================================================
-
-That may occur when using the old list syntax for ``-s`` settings:
-
-::
-
-  # this fails on most Linuxes
-  emcc a.c -sEXPORTED_RUNTIME_METHODS=['foo']
-
-  # this fails on macOS
-  emcc a.c -sEXPORTED_RUNTIME_METHODS="['foo']"
-
-A new, simpler way to specify these lists is to simply use
-comma separated lists:
-
-::
-
-  emcc a.c -sEXPORTED_RUNTIME_METHODS=foo,bar
-
-It is also possible to use a **response file**, that is,
```

Comment:
Is this entry not still useful? I like how it mentions response files in particular.

Reply:
Ok, I'll try to keep that part.

Reply:
Done